include/gridtools/gcl/high_level/numerics.hpp

```diff
@@ -9,8 +9,8 @@
  */
 #pragma once
 
-#include "defs.hpp"
-
 namespace gridtools {
-    constexpr int static_pow3(uint_t i) { return i ? 3 * static_pow3(i - 1) : 1; }
+    namespace gcl {
+        constexpr int static_pow3(int i) { return i ? 3 * static_pow3(i - 1) : 1; }
```

Comment:
```suggestion
        // computes 3^i
        constexpr int static_pow3(int i) { return i ? 3 * static_pow3(i - 1) : 1; }
```
being pragmatic (or find a better name, but probably not worth the effort). (because current name sounds like ^3 to me)